app/stacks/global/front-door/waf-policy.tf

```diff
@@ -590,6 +596,18 @@ resource "azurerm_frontdoor_firewall_policy" "back_office_applications_frontend"
       operator       = "Equals"
       selector       = "description"
     }
+
+    exclusion {
+      match_variable = "RequestBodyPostArgNames"
+      operator       = "Equals"
+      selector       = "descriptionWelsh"
+    }
+
+    exclusion {

```

Comment:
'title' doesn't seem to be included, is it most likely to trigger WAF block for welsh titles ?

Reply:
The error was reported for the Welsh title field because it's Welsh characters like êŵîô that get rejected. I could add it for `title` as well though as that field could in theory contain unicode characters as well.